lib/cdo/pardot.rb

```diff
@@ -64,6 +66,30 @@ class Pardot
   class InvalidApiKeyException < Exception
   end
 
+  # Deletes a list of prospects from Pardot. For Pardot API documentation, see
+  # http://developer.pardot.com/kb/api-version-4/prospects/#using-prospects.
+  # @param [Array[Integer]] prospect_ids of the prospects to be deleted.
+  # @return [Array[Integer]] the set of prospect_ids that failed to be deleted.
+  def self.delete_pardot_prospects(prospect_ids)
```

Comment:
How receptive is Pardot to feature requests? Is it worth our making a feature request for such support?